PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -1,23 +1,92 @@
 <template>
 	<v-container fluid>
+    <v-row justify="center">
+  <v-col class="d-flex" cols="6" sm="6">
+  <v-text-field
+      v-model="qrText"
+      clearable
+      counter
+      filled
+      label="Type some text here"
+      @input="generateQrCode"
+    ></v-text-field>
+    </v-col></v-row>
+
+    <v-row justify="center">
+<div class="justify-center" v-if="qrSrc">
+  <img class="preview" :src="qrSrc"> </div>
+    </v-row>
+
     <v-row>
-QR Code Page
+    <v-btn
+      color="accent"
+      :disabled="!qrSrc"
+      @click="reset"
+    >
+    Reset
+    </v-btn>
+    <v-spacer></v-spacer>
+    <v-btn
+    color="primary"
+      :disabled="!qrSrc"
+      @click="openInNewWindow"
+    >
+    Open 
+    </v-btn>
+
     </v-row>
 	</v-container>
 </template>
 
 <script>
-
+import qrCode from 'qrcode'
```

Comment:
Is this where we typically leave import statements?

Reply:
yes, that's what we do for the event bus. We import it in every page it is used in. Therefore, you need to import it in whatever page the qrcode is used in